src/Sylius/Behat/spec/Context/Setup/PromotionContextSpec.php

```diff
@@ -129,4 +129,21 @@ function it_creates_fixed_discount_promotion_for_cart_with_specified_quantity(
 
         $this->itGivesFixedDiscountToEveryOrderWithQuantityAtLeast('10.00', '5');
     }
+
+    function it_creates_shipping_discount_action_for_promotion(
```

Comment:
I was wondering is it necessary, if "shipping discount" can be only specified with percentage... but probably it's good to point it, as it could different shipping discount action implemented in the future, so :+1: ;)